### PR TITLE
git-tag: add --contains example

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

File: pages/common/git-tag.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Get updated tags from upstream:
 
 `git fetch --tags`
+
+- List tags that contain a given commit:
+
+`git tag --contains `
```
Comment (MarloweW) [inline pages/common/git-tag.md]: --contains should followed by {{commit_id}}.

Comment (waldyrious) [inline pages/common/git-tag.md]: Not sure how I missed this! THanks for the heads-up 👍

File: pages/common/git-tag.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Get updated tags from upstream:
 
 `git fetch --tags`
+
+- List tags that contain a given commit:
```
Comment (waldyrious) [inline pages/common/git-tag.md]: IIUC this lists all tags whose ancestors include the given commit (also, I'm not sure that you can't add more than one tag to a commit). But I'll re-check.

Comment (waldyrious) [inline pages/common/git-tag.md]: Confirmed: http://stackoverflow.com/questions/36246615/how-does-git-tag-contains-work

As for the ability to add multiple tags to a commit, I found nothing suggesting such a restriction in neither https://git-scm.com/book/en/v2/Git-Basics-Tagging nor https://git-scm.com/docs/git-tag.

Comment (agnivade) [inline pages/common/git-tag.md]: >  this lists all tags whose ancestors include the given commit

We should include this then. Makes it clearer.

Comment (waldyrious) [inline pages/common/git-tag.md]: Sure :)

**Pull request merged**